### DISCUSSION TITLE
tailwind css installed but not working in my webpack+ umi + react project

### DISCUSSION THREAD

**Discussion by pangchao21** [tailwindlabs/tailwindcss#13648] [Help]
**What version of Tailwind CSS are you using?**

v3.4.3

**What build tool (or framework if it abstracts the build tool) are you using?**
webpack 

**What version of Node.js are you using?**

v14.16.1

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

Describe your issue

my project use webpack + umi + react (i create project select vanilla js and after add react to it )
i install the tailwind css follow the guide https://tailwindcss.com/docs/guides/vite
no any step is missing , but the tailwindcss seems not work,

config/config.ts
![1715072704885](https://github.com/tailwindlabs/tailwindcss/assets/25843968/f2924c5e-4715-40c3-8e6e-398696d504b2)

install steps
![1715072842358](https://github.com/tailwindlabs/tailwindcss/assets/25843968/2ac087d5-e88a-49c7-abd6-bff915a61003)
![1715072874993](https://github.com/tailwindlabs/tailwindcss/assets/25843968/f5c5cbc2-8ff9-4db7-b725-44e2c5e94c40)
![1715072941087](https://github.com/tailwindlabs/tailwindcss/assets/25843968/3b67abcb-02ac-47ef-bccc-921c6eafa6d1)
![1715073016412](https://github.com/tailwindlabs/tailwindcss/assets/25843968/79155b31-34cf-4a5c-b7bd-ee23017241a6)

**Comment by pxim** [comment]
I have the same problem, tainwind doesn't work in umi 3.x
